python/samples/light/src/bot.py

```diff
@@ -99,12 +102,10 @@ async def on_lights_off(
 
 @app.ai.action("Pause")
 async def on_pause(
-    context: TurnContext,
+    context: ActionTurnContext,
     _state: AppTurnState,
-    data: Any,
```

Comment:
minor: I'm fine with the change. Can you share why you want to remove the data parameter and wrap it into ActionTurnContext? Want to be more clear about the reasons behind the senses.

Reply:
I talked it over with Steve / Kavin a bit last week, my reasoning is in the PR description, it is to simplify handler definitions for the developer and reduce dangling unused parameters. Since python flags the function as an invalid type if we don't include all the parameters, even if the developer doesn't want to use `data`, it might be a bit annoying.

If we wrap it in a new context specific to actions the handlers are simpler and the developer don't need to worry about the function signature as much since all our handlers will be essentially `async Function(TurnContext, State)`